porick/models.py

```diff
@@ -81,12 +82,22 @@ class VoteToUser(db.Model):
     user = relationship("User")
 
 
-class PasswordResets(db.Model):
+class PasswordReset(db.Model):
     __tablename__  = 'password_resets'
     user_id  = Column(Integer, ForeignKey('users.id'), primary_key=True)
-    key = Column(String(26), nullable=False)
+    key = Column(String(32), nullable=False)
     created = Column(DateTime, nullable=False, default=now)
 
+    def __init__(self, user_id):
+        super(PasswordReset, self).__init__()
+        self.user_id = user_id
+        self.key = uuid.uuid4().hex
+
+    @property
+    def is_valid(self):
+        expiry = datetime.timedelta(hours=app.config['PASSWORD_RESET_REQUEST_EXPIRY'])
+        return self.created >= datetime.datetime.now() - expiry
```

Comment:
No I think `>=` is correct. `datetime.datetime.now() - expiry` denotes the earliest possible point in time that the token could have been created and it still be valid (i.e., two hours ago). So, a token is valid if it was created at, or _after_ this time.